api/v1/app.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+""" ssss """
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+from os import getenv
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
+
+
+@app.teardown_appcontext
+def teardown(self):
+    """ sss """
+    storage.close()
+
+
+if __name__ == "__main__":
+    app.run(host=getenv('HBNB_API_HOST', '0.0.0.0'),
+            port=getenv('HBNB_API_PORT', '5000'), threaded=True)
```

Comment:
host and port variables must be inside the script body

Reply:
Done!